inc/header-image-interviewees.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * This is the template the loads the relevant hero image.
+ *
+ * @package MIT_Libraries_Child
+ * @since 2.0.0
```

Comment:
This isn't the Child theme anymore so this should get updated.

Reply:
ok